protobuf/recording.proto

```diff
@@ -78,11 +78,14 @@ message VideoFrameMetaData {
 
 message IMUInfo {
   string gyro_info = 1;
-  float gyro_resolution =2;
+  float gyro_resolution = 2;
   string accel_info = 3;
-  float accel_resolution =4;
-  float sample_frequency = 5; //Hz
-  repeated float placement = 6;
+  float accel_resolution = 4;
+  string mag_info = 5;
+  float mag_resolution = 6;
+
+  float sample_frequency = 7; //Hz
```

Comment:
By changing the digit in the protobuf file we break backwards compatibility.
To ensure that the new reader may operate on old files the digits must be the same for previous fields.

Reply:
ok, nice to know that